src/styles/blocks/page.scss

```diff
@@ -0,0 +1,24 @@
+$day-measure: 100px;
+$gap-size: 1px;
+$days:
```

Comment:
Write variables and functions to a separate file.
Classes for html and body (`page` and `page__container`) in this case it is better not to add them, but to write them to a separate file